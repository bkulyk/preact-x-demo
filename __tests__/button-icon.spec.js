import React from 'react';
import { shallow } from './enzyme';
import ButtonIcon from '@bitchin/button-icon';

it('renders correctly', () => {
  expect(shallow(<ButtonIcon>favorite</ButtonIcon>))
    .toMatchSnapshot();
});

