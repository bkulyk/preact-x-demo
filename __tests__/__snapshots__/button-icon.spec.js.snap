// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Icon
  className="mdc-button__icon"
>
  favorite
</Icon>
`;
